internal/pipeline/executor.go

```diff
@@ -308,8 +308,20 @@ func hydrate(p *pkg, hctx *hydrationContext) (resources []*yaml.RNode, err error
 func filterMetaData(resources []*yaml.RNode) []*yaml.RNode {
```

Comment:
let's rename this function to `filterMetaResources` to reflect what it does.

Reply:
done